packages/kolibri-components/src/KPageContainer.vue

```diff
@@ -47,6 +47,7 @@
 
     padding: 8px 24px 24px;
     margin-top: 24px;
+    overflow-x: scroll;
```

Comment:
This is causing a horizontal scrollbar on all page containers.

Reply:
This might be another case of #5678, with `scroll` working differently on different platforms. In devtools, is there a version of this rule that works for your browser? I could also just move this overflow rule to the wide tables.

Reply:
Well, `scroll` makes the scrollbar always show. On Apple platforms, I think there is styling that essentially makes it invisible, but on FF/Chrome on other platforms it always shows the bulky scrollbar. The value `auto` would be good, but there may be cross-browser inconsistencies. Did you add this primarily for the User table?

Reply:
The UserTable actually already has its own overflow rule. There are other fixed-width very wide, non-table layouts in the app like the Exercise Report that aren't that can overflow the container, so putting this on the KPageContainer is a catch-all.

I'll just change the value to `auto`.

Reply:
Okay sounds good.